### PR TITLE
Create MealFood

### LINKED ISSUE #7: User Story #7 - A user can add food to a meal in the database.
```POST /api/v1/meals/:meal_id/foods/:id```
- [x] A user can add food with :id to the meal with ```:meal_id```.
- [x] A new record is created in the MealFoods table to establish relationship between this food and meal.
- [x] If the meal/food cannot be found, a status code 404 will be returned.
- [x] If successful, a status code of 201 will be returned with the following body: 
```
{
    "message": "Successfully added FOODNAME to MEALNAME"
}
```

### LINKED ISSUE #33: Error in server log on POST new MealFood
Everything seems to be working OK, but the server logs show the following
```
POST /api/v1/meals/1/foods/3 500 65.037 ms - 46
Unhandled rejection Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:470:11)
    at ServerResponse.header (/Users/chakeresa/turing/4module/projects/quantified_self/node_modules/express/lib/response.js:767:10)
    at ServerResponse.send (/Users/chakeresa/turing/4module/projects/quantified_self/node_modules/express/lib/response.js:170:12)
    at Meal.findByPk.then.catch.err (/Users/chakeresa/turing/4module/projects/quantified_self/routes/api/v1/meals.js:62:21)
```

### PR DESCRIPTION
**Pull request by chakeresa**
**Changes proposed in this pull request:**
 - Adds the endpoint to add an existing food to an existing meal: `POST /api/v1/meals/:meal_id/foods/:food_id` (resolves #7)
 - Removed some unused variables
 
**The following checks have been completed:**
 - [x] Tested new feature(s) as well as any feasible edge cases
 - [x] Updated README for changes (new endpoints, new packages, etc)
 - [x] Merged in the latest master & resolved all merge conflicts
 - [x] Ran database migrations
 - [x] Ran the test suite - all tests are passing (or maybe skipped)
 - [x] Checked affected endpoints in Postman
 
**Notes:**
 - Postman was acting a little weird when doing a POST request that should be successful. Fixed if/else -- resolves #33.

### CODE REVIEW

File: routes/api/v1/meals.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-// const Joi = require("@hapi/joi");
 var express = require("express");
 var router = express.Router();
 var Meal = require("../../../models").Meal;
 var MealFood = require("../../../models").MealFood;
 var Food = require("../../../models").Food;
+var MealFood = require("../../../models").MealFood;
```
Comment (chakeresa) [inline routes/api/v1/meals.js]:
```suggestion
```
Duplicate after merge
